sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -184,11 +184,26 @@ private[sql] object Dataset {
  */
 @Stable
 class Dataset[T] private[sql](
-    @transient val sparkSession: SparkSession,
+    @transient val _sparkSession: SparkSession,
```

Comment:
Can this now be `@transient private val`?

Reply:
Done